.circleci/config.yml

```diff
@@ -105,9 +105,11 @@ jobs:
       - STACK_FILE: "stack-9.0.2.yaml"
     <<: *defaults
 
-  ghc-9.2.1:
+  ghc-9.2.2:
     environment:
-      - STACK_FILE: "stack-9.2.1.yaml"
+      # https://github.com/digital-asset/ghc-lib/issues/352
+      - CPATH: "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/ffi:$CPATH"
```

Comment:
The workflow is broken with this: https://github.com/haskell/haskell-language-server/actions/runs/2012818868

Reply:
Oh, apparently I did notice, but this wasn't fixed. It appears to be broken to this day. @pepeiborra do you remember what you did here?

Reply:
I've noticed this and #2936 is attempting to fix it up.